scene/gui/color_picker.cpp

```diff
@@ -1275,51 +1252,36 @@ ColorPicker::ColorPicker() :
 	add_child(memnew(HSeparator), false, INTERNAL_MODE_FRONT);
 
 	VBoxContainer *vbr = memnew(VBoxContainer);
+
 	add_child(vbr, false, INTERNAL_MODE_FRONT);
 	vbr->set_h_size_flags(SIZE_EXPAND_FILL);
 
-	for (int i = 0; i < 4; i++) {
-		HBoxContainer *hbc = memnew(HBoxContainer);
+	GridContainer *gc = memnew(GridContainer);
 
-		labels[i] = memnew(Label());
-		labels[i]->set_custom_minimum_size(Size2(get_theme_constant(SNAME("label_width")), 0));
-		labels[i]->set_v_size_flags(SIZE_SHRINK_CENTER);
-		hbc->add_child(labels[i]);
+	vbr->add_child(gc, false, INTERNAL_MODE_FRONT);
```

Comment:
```suggestion
	vbr->add_child(gc);
```
Doesn't change much in this case, but internal nodes are only useful for direct children of the node (i.e. `add_child()` without `->`).